tests/test_events.py

```diff
@@ -146,7 +145,7 @@ def test_put(self):
                 content_type='application/json'
             )
             self.assertEqual(response.status_code, 200)
-            # TODO: test that the event has a new value
+            self.assertIn(b'new title', response.data)
```

Comment:
Something like `self.assertEqual(flask.json.loads(response.data)['title'], 'new title')` will test the specific field. This will also be easier to read the intent of the test from.